src/app/components/utility/modal/modal.component.ts

```diff
@@ -230,6 +235,31 @@ export class ModalComponent implements OnInit {
     this.denyCallback();
   }
 
+  convertFieldArrayIntoText(fields) {
+    let finalText = '';
+
+    if (fields.length) {
+      if (fields.length >= 1) {
+        if (fields.length > 2) {
+          finalText = `${fields.join(', ')} are not valid. \n\n\nBe sure to include HTTP/HTTPS.`;
+        } else {
+          finalText = `${fields.join(' and ')} are not valid. \n\n\nBe sure to include HTTP/HTTPS.`;
```

Comment:
Hey how about to keep it simple by adding `,` only instead of `and` for the case of length=2 and 3 differently? Like `Google Scholar, GitHub, LinkedIn are not valid. Be sure to include HTTP/HTTPS.` or `Google Scholar, GitHub are not valid. Be sure to include HTTP/HTTPS.`